cloudevents-sdk-reqwest/src/client_response.rs

```diff
@@ -107,6 +108,19 @@ pub async fn response_to_event(res: Response) -> Result<Event> {
     MessageDeserializer::into_event(ResponseDeserializer::new(h, b))
 }
 
+/// Extention Trait for [`Response`]
+#[async_trait]
+pub trait ResponseExt {
+    async fn to_event(self) -> Result<Event>;
```

Comment:
I think it's better to name this method `into_event`. In Rust conventionally `to_*` methods takes a ref to self, while `into_*` takes ownership (like this one)